tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -9,3 +9,19 @@ spec:
     SESSION_TIMEOUT = 3600
 status:
   readyCount: 1
+---
+# Test the status code is correct for each endpoint
+# This test is for heat endpoints
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: |
+      set -x
+      RETURN_CODE=0
+      PUBLIC_URL=$(oc get -n $NAMESPACE horizon horizon -o jsonpath='{.status.endpoint}')
+      STATUSCODE=$(curl --silent --output /dev/stderr --head --write-out "%{http_code}" $PUBLIC_URL/dashboard/auth/login/?next=/dashboard/")
```

Comment:
Looks like you should add `"` before `$PUBLIC_URL` ?

Reply:
Ah yeah, I see where I missed it. Fixed in the latest revision.